openfl/federated/task/metric.py

```diff
@@ -0,0 +1,11 @@
+"""Metric module."""
+
+from typing import NamedTuple
+import numpy as np
+
+
+class Metric(NamedTuple):
```

Comment:
The definition for `Metric` should probably be kept in `openfl.utils.types` where `TensorKey` and `TaskResultKey` is defined